src/objecttypes/setup_configuration/tests/test_token_auth_config.py

```diff
@@ -0,0 +1,441 @@
+        }
+        with self.assertRaises(ConfigurationRunFailed) as command_error:
+            setup_config = build_step_config_from_sources(
+                TokenAuthConfigurationStep,
+                object_source=object_source,
+            )
+            step = TokenAuthConfigurationStep()
+            step.execute(setup_config)
+
+        self.assertTrue(
+            "Failed configuring token token-2" in str(command_error.exception)
+        )
+        self.assertEqual(TokenAuth.objects.count(), 1)
+
+    def test_invalid_setup_contact_person(self):
+        object_source = {
+            "objecttypes_tokens_config_enable": True,
+            "objecttypes_tokens": {
+                "items": [
+                    {
+                        "identifier": "token-1",
+                        "token": "ba9d233e95e04c4a8a661a27daffe7c9bd019067",
+                        "contact_person": "",
+                        "email": "person-1@example.com",
+                        "organization": "Organization 1",
+                        "application": "Application 1",
+                        "administration": "Administration 1",
+                    },
+                ],
+            },
+        }
+        setup_config = build_step_config_from_sources(
+            TokenAuthConfigurationStep,
+            object_source=object_source,
+        )
+        with self.assertRaises(ConfigurationRunFailed) as command_error:
+            step = TokenAuthConfigurationStep()
+            step.execute(setup_config)
+
+        self.assertTrue(
+            "Validation error(s) occured for token-1" in str(command_error.exception)
+        )
+        self.assertEqual(TokenAuth.objects.count(), 0)
+
+    def test_invalid_setup_identifier(self):
+        object_source = {
+            "objecttypes_tokens_config_enable": True,
+            "objecttypes_tokens": {
+                "items": [
+                    {
+                        "identifier": "invalid identifier",
+                        "token": "ba9d233e95e04c4a8a661a27daffe7c9bd019067",
+                        "contact_person": "Person 1",
+                        "email": "person-1@example.com",
+                        "organization": "Organization 1",
+                        "application": "Application 1",
+                        "administration": "Administration 1",
+                    },
+                ],
+            },
+        }
+        setup_config = build_step_config_from_sources(
+            TokenAuthConfigurationStep,
+            object_source=object_source,
+        )
+        with self.assertRaises(ConfigurationRunFailed) as command_error:
+            step = TokenAuthConfigurationStep()
+            step.execute(setup_config)
+        self.assertTrue(
+            "Validation error(s) occured for invalid identifier"
+            in str(command_error.exception)
+        )
+        self.assertEqual(TokenAuth.objects.count(), 0)
+
+    def test_valid_call_command(self):
```

Comment:
I think this test can be removed, since this functionality is already tested in the `django-setup-configuration` library